scripts/o3de/o3de/manifest.py

```diff
@@ -656,60 +656,72 @@ def get_registered(engine_name: str = None,
         for project_path in projects:
             project_path = pathlib.Path(project_path).resolve()
             project_json = project_path / 'project.json'
-            with project_json.open('r') as f:
-                try:
-                    project_json_data = json.load(f)
-                except json.JSONDecodeError as e:
-                    logger.warning(f'{project_json} failed to load: {str(e)}')
-                else:
-                    this_projects_name = project_json_data['project_name']
-                    if this_projects_name == project_name:
-                        return project_path
+            if not pathlib.Path(project_json).is_file():
+                logger.warning(f'{project_json} does not exist')
+            else:
+                with project_json.open('r') as f:
+                    try:
+                        project_json_data = json.load(f)
+                    except json.JSONDecodeError as e:
+                        logger.warning(f'{project_json} failed to load: {str(e)}')
```

Comment:
nit: For python f-strings, '{}' does an implicit string conversion anyways, so no need for str(e). (same as the others below)

Reply:
Thanks! I want to keep this PR as a clean merge though so don't want to introduce any modifications. This can be handled in a follow-up PR.